mava/utils/loggers/base.py

```diff
@@ -173,13 +186,20 @@ def make_logger(
         return logger
 
     def _path(self, subdir: Optional[str] = None) -> str:
-        if subdir:
-            path = str(self._directory / self._time_stamp / subdir / self._label)
+        if isinstance(self._directory, str):
+            if subdir:
+                path = f"{self._directory}/{self._time_stamp}/{subdir}/{ self._label}"
+            else:
+                path = f"{self._directory}/{self._time_stamp}/{self._label}"
+            return path
         else:
-            path = str(self._directory / self._time_stamp / self._label)
+            if subdir:
+                path = str(self._directory / self._time_stamp / subdir / self._label)
+            else:
+                path = str(self._directory / self._time_stamp / self._label)
```

Comment:
not following what's happening here, if it's not a string what is it and what does the divide operator do?

Reply:
So the divider does a similiar thing to the string case. You just can't do `self._directory / ...` with strings (without f strings). Here for a path, you can do this without converting the path object to a string and a couple lines below we do some path cleanup that ensures it is a valid path (`paths.process_path`).

Reply:
Ok nice